src/ui/zcl_abapgit_gui_page_syntax.clas.abap

```diff
@@ -32,9 +32,9 @@ CLASS ZCL_ABAPGIT_GUI_PAGE_SYNTAX IMPLEMENTATION.
 
     DATA: li_syntax_check TYPE REF TO zif_abapgit_code_inspector.
 
-    li_syntax_check = zcl_abapgit_factory=>get_syntax_check( mo_repo->get_package( ) ).
+    li_syntax_check = zcl_abapgit_factory=>get_code_inspector( mo_repo->get_package( ) ).
 
-    mt_result = li_syntax_check->run( ).
+    mt_result = li_syntax_check->run( 'SYNTAX_CHECK' ).
```

Comment:
What do you think about ectracting this to a constants structure? In that way we can easily keep track of the variants we use.

Reply:
*shrug*